packages/neuron-ui/src/utils/const.ts

```diff
@@ -4,8 +4,29 @@ export enum NetworkStatus {
 }
 
 export enum Channel {
-  SendCapacity = 'sendCapacity',
+  // Chain
+  GetBlock = 'getBlock',
+  GetTransaction = 'getTransaction',
+  GetLiveCell = 'getLiveCell',
+  GetTipHeader = 'getTipHeader',
+  GetTipBlockNumber = 'getTipBlockNumber',
+  GetLocalNodeId = 'getLocalNodeId',
+  GetNetwork = 'getNetwork',
+  GwitchNetwork = 'switchNetwork',
```

Comment:
```suggestion
  SwitchNetwork = 'switchNetwork',
```

Reply:
This could be my typo in previous PR suggestions.

Reply:
Fixed. (Need to fix that enum in `neuron-wallet` as well).

Reply:
Also updated on `develop`. Please merge `nervosnetwork/neuron/develop` to resolve conflicts.